specs/casper_sharding_v2.1.md

```diff
@@ -298,15 +298,18 @@ def get_block_hash(active_state, curblock, slot):
 
 `get_block_hash(*, *, h)` should always return the block in the chain at slot `h`, and `get_shards_and_committees_for_slot(*, h)` should not change unless the dynasty changes.
 
-Finally, we abstractly define `integer_sqrt` for use in reward/penalty calculations:
+Finally, we abstractly define `integer_sqrt(x)` for use in reward/penalty calculations as the largest integer `n` such that `n**2 <= x`. Here is one possible implementation, though clients are free to use their own including standard libraries if available. 
```

Comment:
It will better if swapping the notations to match the code:
the largest integer `x` such that `x**2 <= n`

Reply:
actually pushing up an implementation that satisfies the criteria. I'll use your language here though. It's good